packages/SwingSet/src/vats/comms/dispatch.js

```diff
@@ -18,7 +18,7 @@ export function buildCommsDispatch(
   _vatPowers,
   vatParameters = {},
 ) {
-  const { identifierBase = 0 } = vatParameters;
+  const { identifierBase = 0, useSeqNums = true } = vatParameters;
```

Comment:
Should we call this `sendExplicitSeqNums`? We're always *using* seqnums, each outgoing message is always associated with a sequence number, but the question is whether we make that number explicit, or if we have the receiver rely solely upon counting messages.

Reply:
Good point.  I will make this change.